L1Trigger/L1TMuonBarrel/interface/L1TMuonBarrelParamsHelper.h

```diff
@@ -23,8 +23,10 @@
 #include "L1Trigger/L1TCommon/interface/Parameter.h"
 #include "L1Trigger/L1TCommon/interface/Mask.h"
 
-#include "CondFormats/L1TObjects/interface/L1MuDTQualPatternLut.h"
-#include "CondFormats/L1TObjects/interface/L1MuDTEtaPatternLut.h"
+//#include "CondFormats/L1TObjects/interface/L1MuDTQualPatternLut.h"
```

Comment:
No.  They are not.  They are still used by L1TriggerConfig/DTTrackFinder.
Will remove commented lines.